source/mig.md

```diff
@@ -0,0 +1,76 @@
+# Multi-Instance GPU (MIG)
+
+Multi-Instance GPU is a technology that allows partitioning a single GPU into multiple instances, making each one seem as a completely independent GPU. Each instance then receives a certain slice of the GPU computational resources and a pre-defined block of memory that is detached from the other instances by on-chip protections.
+
+Due to the protection layer to make MIG secure, certain limitations exist. One such limitation that is generally important for HPC applications is the lack of support for CUDA Inter-Process Communication (IPC), which enables transfers over NVLink and NVSwitch to greatly speed up communication between physical GPUs. When using MIG, NVLink and NVSwitch are thus completely unavailable, forcing the application to take a more expensive communication channel via the system (CPU) memory.
```

Comment:
Could we link "CUDA Inter-Process Communication (IPC)" somewhere useful? And maybe "NVLink and NVSwitch"?

Reply:
Done in https://github.com/rapidsai/deployment/pull/63/commits/19c6a4169bc7ae9a262ca2188a82dedf0e3f0ff4 .